apps/console/src/extensions/configs/scim.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { SCIMConfigInterface } from "./models/scim";
+
+export const SCIMConfigs: SCIMConfigInterface = {
```

Comment:
Shall we add a small doc comment explaining this config.

Reply:
Added in commit 709db97832e822082dff6db0d23962b5e6773072